keyboards/splitography/config.h

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2021 Alexis Jeandeau
```

Comment:
The PR guidelines indicate that a license header is required for all .c and .h source files, so I added this since the original files don't have license information and don't mention an author.
Should I mention @sdothum and @analyticd?